### ISSUE TITLE
[refactor] Remove placeholder in GetStartedPosition

### ISSUE THREAD

**Issue by szpasztor** [decred/decrediton#2032]
Is there a purpose of [this](https://github.com/decred/decrediton/blob/master/app/components/views/GetStartedPosition.js#L23) "sadfsdf"?

**Comment by alexlyp** [comment]
Nope, but I believe it was throwing some errors due to using an empty/null tag so I threw in some text to please it.
